nmdc_runtime/site/translation/neon_soil_translator.py

```diff
@@ -116,7 +134,6 @@ def _translate_biosample(
         """
         return nmdc.Biosample(
             id=nmdc_id,
-            part_of="nmdc:sty-11-34xj1150",
```

Comment:
`part_of` should be replaced with `associated_studies` instead of just removing this line.